app/src/main/res/values/styles.xml

```diff
@@ -95,6 +95,17 @@
     <item name="android:progressDrawable">@drawable/progress_bar_green_horizontal</item>
   </style>
 
+  <style name="AlertDialog" parent="Theme.AppCompat.Light.Dialog.Alert">
+    <item name="buttonBarPositiveButtonStyle">@style/AlertDialogButtonStyle</item>
+    <item name="buttonBarNegativeButtonStyle">@style/AlertDialogButtonStyle</item>
+    <item name="colorAccent">@color/text_secondary</item>
+  </style>
+
+  <style name="AlertDialogButtonStyle" parent="Widget.AppCompat.Button.ButtonBar.AlertDialog">
+    <item name="android:textColor">@color/ksr_green_500</item>
```

Comment:
Should we refer to this is `colorAccent`?

Reply:
So it was ignoring the color when I used colorAccent so I went with TextColor

Reply:
Sorry, I wasn't clear. I mean the color value you're passing in. Like this `<item name="android:textColor">@color/accent</item>`